app/Controllers/Http/EventsController.ts

```diff
@@ -1,5 +1,6 @@
 import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
 import Event from 'App/Models/Event'
+import NewEventValidator from 'App/Validators/NewEventValidator'
```

Comment:
Ich würde den in `CreateEventController` umbenennen. new ist ja ein kexword und könnte verwirren?

Reply:
ja das macht sinn

Reply:
Kannst du dann auch gleich einen validator für `updateEvent` machen? 

bei dem kann man ja das `is_public` field ändern. 
Gerne aber auch in einem neuen PR. Oder ich mache das :woman_shrugging: